src/selectedtests/work_items/cli.py

```diff
@@ -7,49 +7,116 @@
     process_queued_test_mapping_work_items,
     process_queued_task_mapping_work_items,
 )
-from selectedtests.helpers import get_evg_api, get_mongo_wrapper, setup_logging
+from selectedtests.datasource.mongo_wrapper import MongoWrapper
+from selectedtests.helpers import get_evg_api, setup_logging
+from selectedtests.evergreen_helper import get_evg_project
+from selectedtests.work_items.project_task_mapping_work_item import ProjectTaskMappingWorkItem
+from selectedtests.work_items.project_test_mapping_work_item import ProjectTestMappingWorkItem
+
+DEFAULT_WEEKS_BACK = 24
 
 
 @click.group()
 @click.option("--verbose", is_flag=True, default=False, help="Enable verbose logging.")
+@click.option("--mongo-uri", required=True, type=str, help="Mongo URI to connect to.")
 @click.pass_context
-def cli(ctx, verbose: str):
+def cli(ctx, verbose: str, mongo_uri: str):
     """Entry point for the cli interface. It sets up the evg api instance and logging."""
     ctx.ensure_object(dict)
-    ctx.obj["mongo"] = get_mongo_wrapper()
+    ctx.obj["mongo"] = MongoWrapper.connect(mongo_uri)
     ctx.obj["evg_api"] = get_evg_api()
 
     setup_logging(verbose)
 
 
 @cli.command()
+@click.option("--project", type=str, required=True, help="Evergreen project to add.")
```

Comment:
I added this command and a similar command below for task mapping. They both add an item to the appropriate queue to be processed. I find this convenient for local testing. I can add a project to the queue and then run the worker to process that item.

Reply:
Makes sense 👍